### PR TITLE
TW-2085 Change texts in settings screen

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
 #2085 
## Impact description

- [x]  Change text to “Disable notifications” when the toggle is on and “Enable notifications” when the toggle is off 
- [x]  Change colour of the headers in Notifications screen
- [x]  Remove description in app language
- [x]  Remove all the descriptions for blocks in Settings
- [x]  Change name of the block to just “Notifications”

## Resolved
- Web:

https://github.com/user-attachments/assets/a2354a40-85c9-46cd-b548-872ddec5997f

### CODE REVIEW

**Comment by nqhhdev** [comment]
hi @KhaledNjim should resolve conflict

**Comment by nqhhdev** [comment]
Hi @KhaledNjim, pls help me check it ... haizzzz This is such a simple thing, how can you let it be so broken? Don't waste the reviewer's time.

Screenshot 2024-10-16 at 13 20 10

**Comment by hoangdat** [comment]
image
@KhaledNjim please keep title `Notifications` in Center, the same for other settings

**Comment by KhaledNjim** [comment]
> * [x]  changing icon app in appbar of web app only

image

File: assets/l10n/intl_en.arb [hunk 1]
```diff
@@ -1350,11 +1350,6 @@
     "type": "text",
     "placeholders": {}
   },
-  "notificationsEnabledForThisAccount": "Notifications enabled for this account",
-  "@notificationsEnabledForThisAccount": {
-    "type": "text",
-    "placeholders": {}
-  },
```
Comment (hoangdat) [inline assets/l10n/intl_en.arb:1357]: need to delete in other languages?

File: assets/l10n/intl_en.arb [hunk 2]
```diff
@@ -2847,20 +2842,6 @@
...
-  "settingsNotificationAndSoundsSubtitle": "Custom how you get notifications from Twake such as previewing messages, sounds, time,...",
-  "@settingsNotificationAndSoundsSubtitle": {},
-  "settingsChatFoldersSubtitle": "Create and manage folders for different groups of chats and quickly switch between them.",
-  "@settingsChatFoldersSubtitle": {},
-  "settingsAppLanguageSubtitle": "Choose in which language you want to use Twake Chat.",
-  "@settingsAppLanguageSubtitle": {},
-  "settingsDevicesSubtitle": "Control your sign in and sign out on any device.",
-  "@settingsDevicesSubtitle": {},
-  "settingsHelpSubtitle": "Help center, contact us, privacy policy.",
-  "@settingsHelpSubtitle": {},
```
Comment (hoangdat) [inline assets/l10n/intl_en.arb:2863]: idem

Comment (KhaledNjim) [reply assets/l10n/intl_en.arb:2863]: removed

**Review by hoangdat — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**